critiquebrainz/frontend/templates/macros.html

```diff
@@ -220,3 +220,40 @@ <h4 style="margin-bottom:0;">{{ _('Rate this {}:'.format(entity_type_readable))
         -
     {% endif %}
 {% endmacro %}
+
+{% macro display_pagination(current_page, total_pages, page_info, parameter) %}
```

Comment:
It looks like `count/limit|round(method='ceil')|int` theme is common in callers of this macro. I suggest you take count and limit as parameters to this macro. This way you can fold the common if check as well into here and also apply the modification at one place.